instrumentation/jms/src/main/java/brave/jms/TracingConsumer.java

```diff
@@ -62,6 +63,9 @@ void handleReceive(Message message) {
       span.start(timestamp).finish(timestamp);
     }
     injector.inject(span.context(), request);
+    if (message instanceof BytesMessage) {
+      (BytesMessage message).reset();
```

Comment:
@me3x sorry if I wasn't clear enough. This part is not compiling, and actually not needed. I believe jms will take care of the message itself; no need to reset ourselves. I've give a try removing this and verifying ITs locally and all's green :)

Reply:
Instrumentation should left the message in the same state it started. I tried to read body in JmsListener and it failed.